packages/web/config/price.ts

```diff
@@ -874,6 +874,19 @@ export const PoolPriceRoutes: IntermediateRoute[] = [
     ),
     destCoinId: "pool:uluna",
   },
+  {
+    alternativeCoinId: "pool:staevmos",
+    poolId: "", // <= fill this out once pool is live
+    spotPriceSourceDenom: DenomHelper.ibcDenom(
+      [{ portId: "transfer", channelId: "channel-326" }],
+      "stuluna"
```

Comment:
when you update the pool id, don't forget to correct this denom

Reply:
Nice catch @JeremyParish69 . Will do, thank you!